### DISCUSSION TITLE
invalid - api key , how to solve this problem

### DISCUSSION THREAD

**Discussion by singghen** [MuiseDestiny/zotero-gpt#252] [Q&A]
Hi~~
I downloaded ZOTERO GPT from the website. At first, I looked at the netizen's teaching, using my own original openai generated API KEY and later fount that when I wanted to use the ZOTERO translate, it would say that 'You exceeded your current quota, please check your plan and billing details'

So I use the internet free API- recommended by nitizens 

https://github.com/chatanywhere/GPT_API_free

However, when I update the new API KEY and want to translate the PDF file,
It will say 'invalid-api key', asking me to go back to openai and get the API-KEY for my original account. But the APIKEY of my original account can not be translated, that's why I hope to use the free APIKEY recommended by the network! How can I solve it? 
thank you very much

**Comment by abu23hao** [comment]
I have the same question. Do you have to solve it？

**Comment by singghen** [comment]
No, I forgot it. Could you please send me the link? Thanks

abu23hao ***@***.***>於 2024年5月10日 週五，10:26寫道：

**Reply by abu23hao** [reply]
I have solve it. You can look at this  web.   https://www.bilibili.com/video/BV17N4y1o7vx/?spm_id_from=333.337.search-card.all.click&vd_source=9358bc6a7695f6cc0e77c8a5cdf8e1fc

**Comment by singghen** [comment]
Thank you very much 😊

abu23hao ***@***.***>於 2024年5月11日 週六，09:32寫道：
